interchain-exchange/tutorial/00-intro.md

```diff
@@ -5,7 +5,7 @@ order: 1
 # Introduction
 
 The Interchain Exchange is a module to create buy and sell orders in between blockchains.
-In this tutorial you will learn how to create a Cosmos SDK module which can create order pairs, buy and sell orders. You will be able to create orderbooks, buy and sell orders across blockchains, which enables to swap tokens from one blockchain to another.
+In this tutorial you will learn how to create a Cosmos SDK module which can create order pairs, buy and sell orders. You will be able to create orderbooks, buy and sell orders across blockchains.
```

Comment:
```suggestion
In this tutorial, you will learn how to create a Cosmos SDK module that can create order pairs, buy and sell orders. You will be able to create order books, buy and sell orders across blockchains.
```

Reply:
We always use `orderbook` as one-word. I would like to stay consistent with this wording.

Reply:
Shouldn't we change it everywhere in the future? I mean I don't think orderbook is grammatically correct

Reply:
We can use `order book` and then consistent. I can make a PR with that change tomorrow